### DISCUSSION TITLE
Uninstaller for Safe Exam Browser

### DISCUSSION THREAD

**Discussion by tjpete** [SafeExamBrowser/seb-mac#301] [General]
Hello, is there an uninstaller for the SEB or documentation to manually uninstall? 

Thanks,

**Comment by danschlet** [comment]
No, as there is no installer, there's also no uninstaller. You can just remove the Safe Exam Browser app from the /Applications folder. SEB doesn't save anything into other locations, just macOS creates `org.safeexambrowser.SafeExamBrowser.plist` in ~/Library/Preferences/ as usual for macOS apps.
To remove settings completely, you though need to use this command in Terminal (mentioned in manual and FAQ):
`defaults delete org.safeexambrowser.SafeExamBrowser `

**Comment by tjpete** [comment]
Sorry I do have one more issue.  Setting the password for the
Settings-Preferences so students can't manipulate. I tried both password
options in the config but it then requires a student to enter the password
before the browser will open.

Thanks,

On Mon, May 8, 2023 at 3:32 PM Daniel R. Schneider ***@***.***>
wrote:

> Closed #301 <https://github.com/SafeExamBrowser/seb-mac/discussions/301>
> as resolved.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SafeExamBrowser/seb-mac/discussions/301>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEUJBQVSNZW4UTW5KI4QJFTXFFJ5JANCNFSM6AAAAAAXU2GRGM>
> .
> You are receiving this because you authored the thread.Message ID:
> <SafeExamBrowser/seb-mac/repo-discussions/301/discussion_event/652950@
> github.com>
>


-- 
Teresa Peterson
Hampton Dumont Technology Department
601 12th Ave NE
Hampton, IA 50441

Office: 641-456-4735 Ext 119
Fax: 641-456-2023

This message is intended only for the use of the Addressee and may contain
information that is PRIVILEGED and CONFIDENTIAL. If you are not the
intended recipient, dissemination of this communication is prohibited. If
you have received this communication in error, please erase all copies of
the message and its attachments and notify the sender immediately. Thank
you.
